.travis.yml

```diff
@@ -8,24 +8,37 @@ language: ruby
 # We use ruby 1.9.3p374; available versions are:
 # ruby-1.9.3-p547/ruby-1.9.3-p545-railsexpress/ruby-1.9.3-p545/ruby-1.9.3-p484
 rvm:
-  - 1.9.3-p547
   - 2.1.2
 
 # Environment Variables
-# - DB=sqlite
-# - DB=postgresql # This is needed instead of mysql for Heroku
 env:
-  - DB=mysql
+  global:
+    - RAILS_ENV=test
+    - TZ=America/New_York # Set the system Time Zone to be EST
+
+# Dependencies
+before_install:
+  - sudo apt-get install -y imagemagick qt-sdk
```

Comment:
ImageMagick already ships in the Travis container so we don't need to install it